Linux/flash

```diff
@@ -171,43 +171,17 @@ case "${OSTYPE}" in
 esac
 
 while true; do
-  if [[ -z "${disk}" ]]; then
-    # try to find the correct disk of the inserted SD card
-    disk=$(df | grep /media | cut -f1 -d " " | sed -e 's/[0-9]$//'|sed  -e 's/p.*$//' | sort | uniq)
-
-    if [[ $(echo "${disk}"|awk '{print NF}') -gt 1 ]]; then
```

Comment:
@StefanScherer: Unfortunately, I don't see a way to keep that logic for systems that mount in /media without also confusing users of non-automounting systems. The problem is that if the system is not configured to automount into /media (like mine), then the program will suggest that there's no SD card inserted, and the user will be confused when inserting an SD card doesn't fix anything. Adding to the complexity, some systems are configured to automount but into somewhere besides /media, or to automount different device types into different locations. There's not even an easy way to detect whether or not a system is configured to do automounting, or the exact behavior it's configured for. In short, I don't think there's a uniform way to make this logic work everywhere.

If you can describe the logic you'd like to follow on both automounting and non-automounting systems, I can turn it into a patch. However, right now, I'm not seeing a way of handling all the cases.